README.md

```diff
@@ -22,6 +22,6 @@ Please note that this *does not cover* the usage of the "osu!" or "ppy" branding
 
 Contributions to the osu!wiki are now done through GitHub.
 
-A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [here](/wiki/osu!wiki_contribution_guide).
+A basic guide to using GitHub, specifically tailored for the osu!wiki project, is available [](/wiki/osu!wiki_contribution_guide).
```

Comment:
This will remove the link overall!
Use `[osu!wiki contribution guide](/wiki/osu!wiki_contribution_guide).`